cmd/digdirator/main.go

```diff
@@ -41,7 +42,12 @@ func init() {
 	_ = nais_io_v1.AddToScheme(scheme)
 	// +kubebuilder:scaffold:scheme
 
-	log.SetLevel(log.DebugLevel)
+	lvl, err := log.ParseLevel(viper.GetString(config.LogLevel))
+	if err != nil {
+		log.SetLevel(log.InfoLevel)
+	} else {
+		log.SetLevel(lvl)
+	}
```

Comment:
Looks like the if and else here are flipped

Reply:
Wait, nevermind - it was correct. Too early for me 😄

Reply:
haha :D

Reply:
Skal lage en annen bedre if